### DISCUSSION TITLE
Music Brainz invalid ID

### DISCUSSION THREAD

**Discussion by rvandet** [music-assistant/support#1682] [Q&A]
Error in Log: "Invalid MusicBrainz identifier"

How can i prevent this, it is slowing down my system and it seemily prevents albums showing tracks.

**Comment by OzGav** [comment]
If you can find which track it is then fix it. With the limited info you have provided above I am guessing there is a tag problem…?

**Comment by aaronk6** [comment]
@rvandet Did you tag your files with beets by any chance? I know beets is writing Spotify tags to MusicBrainz fields, which will (rightfully) cause the “Invalid MusicBrainz identifier” in Music Assistant.

**Comment by rvandet** [comment]
I use MP3Tag

**Reply by aaronk6** [reply]
Ah, I see. Do you still have the problem though?

**Comment by rvandet** [comment]
Not  that i am aware of.

**Comment by schnitzi99** [comment]
I experienced the same problem and in my case it seems like there is a special character at the end of the music brainz track id.
This leads to the tracks / albums not appearing in my library - even with version 2.8.5 of music assistant.
Automatically retagging the files with a recent version of MusicBrainz Picard solved the issue for me.
A list of the files was available in music assistant, settings, system, background tasks.

**Comment by iophobia** [comment]
Sorry to flog this dead horse/topic but I'm facing the same issue despite a normally well-tagged library. Is there any possibility to include the storage path or at least filename for the errors in question? Because for some wrong MBIDs I don't even know where to start since I can't find anything online or in the logs.

**Reply by OzGav** [reply]
You will need to provide more information for us to help

**Reply by iophobia** [reply]
Example from background (album) syncing tasks:
`2026-07-29 08:00:27 INFO [music_assistant.tasks] Task started
2026-07-29 08:17:17 ERROR [music_assistant.tasks] Task failed: Invalid MusicBrainz identifier: D16d93e8-61d9-4222-9620-7f8a1fdb42cd
2026-07-29 08:17:17 WARNING [music_assistant.tasks] Background task Sync OpenSubsonic Media Server Library albums failed: Invalid MusicBrainz identifier: D16d93e8-61d9-4222-9620-7f8a1fdb42cd`

How do I find the file that needs re-tagging?

**Reply by OzGav** [reply]
It has a capital D. I have a PR in for a different problem that will improve the logging. After that is merged we can look at extending to cover the opensubsonic provider

**Reply by OzGav** [reply]
OK PR has been opened https://github.com/music-assistant/server/pull/5149
